data/styles/clear/include/Icons.mapcss

```diff
@@ -2091,6 +2091,13 @@ node|z17-[amenity=parcel_locker],
 area|z17-[amenity=parcel_locker],
 {icon-image: mail-m.svg;icon-min-distance: 20;}
 
+node|z17[amenity=vending_machine][vending=fuel],
+area|z17[amenity=vending_machine][vending=fuel]
+{icon-image: zero-icon.svg;text: none;}
```

Comment:
Why zero icon is needed on z17?

Reply:
Because we have generic vending_machine icon, inherited from amenity=vending_machine, that draw in z17.